projects/openapi-generator.tech/package.yml

```diff
@@ -0,0 +1,59 @@
+distributable:
+  url: git+https://github.com/OpenAPITools/openapi-generator.git
+  ref: ${{version.tag}}
+
+versions:
+  github: OpenAPITools/openapi-generator
+
+dependencies:
+  openjdk.org: ^20
+
+build:
+  dependencies:
+    maven.apache.org: '*'
+  script:
+    - mvn clean package -Dmaven.javadoc.skip=true
+    - install -D modules/openapi-generator-cli/target/openapi-generator-cli.jar {{prefix}}/libexec/lib/openapi-generator-cli.jar
+    - run: |
+        cat > openapi-generator <<EOF
+        #!/bin/sh
+        exec java -jar \$(dirname \$0)/../libexec/lib/openapi-generator-cli.jar "\$@"
+        EOF
+        chmod +x openapi-generator
+      working-directory: ${{prefix}}/bin
```

Comment:
this can also be:
```yaml
  - run: install $PROP openapi-generator
    prop: |
      #!/bin/sh
      exec java -jar $(dirname $0)/../libexec/lib/openapi-generator-cli.jar "$@"
    working-directory: ${{prefix}}/bin
```
i believe, if you find that simpler. (double check the output, though. i think we removed the need to escape `$`, but i'm not _positive_.)